Objective-C/Internal/Replicator/CBLWebSocket.mm

```diff
@@ -41,8 +41,10 @@
 
 static constexpr size_t kMaxReceivedBytesPending = 100 * 1024;
 static constexpr NSTimeInterval kConnectTimeout = 15.0;
-static constexpr NSTimeInterval kIdleTimeout = 300.0;
 
+// The value should be greater than the heartbeat to avoid read/write timeout;
+// the current default heartbeat is 300 sec:
+static constexpr NSTimeInterval kIdleTimeout = 320.0;
```

Comment:
The heartbeat happens down inside LiteCore and isn't known at this level. CBLWebSocket just gets told to send PING messages periodically.